item.html

```diff
@@ -0,0 +1,113 @@
+<DOCKTYPE html>
+<html>
+    <head>
+        <meta charset="uts-8">
+        <title>Bsxter of california</title>
+    </head>
+    <body>
+        <header>
+            <nav class="main-navigation">
+                <a class="main-page">
+                    <img scr="barbershop-logo" href="index.html" alt="Барбершоп Бородинский">
+                </a>
+                <ul class="site-navigation">
+                    <li><a href="#">Информация</a></li>
+                    <li><a href="#">Новости</a></li>
+                    <li><a href="#">Прайс-лист</a></li>
+                    <li><a href="#">Магазин</a></li>
+                    <li><a href="#">Контакты</a></li>
+                </ul>
+                <ul class="user-navigation">
+                    <li class="login-link">
+                        <a class="login-link" href="login.html">Вход</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <ul class="shop-navigation">
+                <li>Главная</li>
+                <li>Магазин</li>
+                <li>Средства для ухода</li>
+                <li>Набор для путешествий "Bsxter of California"</li>
+            </ul>
+            <section>
+                <img src="#">
+                <ul>
+                    <li>
+                        <p>есть в наличии</p>
+                        <p>Артикул: Dexter-AE</p>
+                    </li>
+                    <li>
+                        <p>Travel kit - необходимый аксессуар во время любого путешествия. В аккуратной кожанной сумке находится все, что нужно для бритья и ухода за кожей во время рабочей поездки или отдыха: средство для умывания, увлажняющий крем, крем для бритья, крем после бритья, шампунь. Набор также может стать отличныи подарком</p>
+                    </li>
+                    <span class="value-info"></span>
```

Comment:
```suggestion
                    <li><span class="value-info"></span></li>
```

внутри тега ul может быть только li, здесь и далее